README.md

```diff
@@ -87,8 +86,9 @@ For Shesmu to know what actions it can perform, it uses an action repository. A
 new action repository can be created using the `ActionRepository` interface.
 The following are available:
 
-- JSON-over-HTTP interface. To define, add to the `SHESMU_ACTION_URLS`
-  environment variable. See [the remote action repository specification](api.md).
+- JSON-over-HTTP interface. To add one, create a JSON file ending in `.remote`
+  in the `SHESMU_DATA` directory containing `{ "url": ... }`. See
```

Comment:
It could be clearer here that the file should contain `{ "url": ... }`, and not the directory...